### ISSUE TITLE
Building envelope updated U-values study (Oct 2018)

### ISSUE THREAD

**Issue by IainMacdonald** [canmet-energy/btap_tasks#192]
### Description
PAT project containing limited set of archetypes and locations to test suggested updates to U-values in NECB tables

### Approach
Using measure created in task #130. 

### Waiting On
SC-EE requires NECB 2017 as baseline. Currently using 2015. 

See doc for values to simulate:
[Building Envelope Task Group - updates to U-values.docx](https://github.com/canmet-energy/btap_tasks/files/2626462/Building.Envelope.Task.Group.-.updates.to.U-values.docx)

**Comment by IainMacdonald** [comment]
TO DO
Add task to NZER project
Upload PAT model to nrc_192 branch (Edward's fork)
NRC and NRCan to run and confirm agreement on numbers
When available re-run with 2017 as baseline.

**Comment by phylroy** [comment]
waiting on items from NRC.

**Comment by edwardv720** [comment]
PAT projects to run these simulations for 2017 - https://github.com/edwardv720/btap/tree/nrc_192/pat_projects/TG-BE%20envelope%20improvement%20(Uvalue)

NECB2017 folder - baseline sims
NECB2017 Window improvement folder  - options for windows
NECB2017 opaque u-value improvements folder  - options for opaque elements

**Comment by phylroy** [comment]
[NECB2017simulations.zip](https://github.com/canmet-energy/btap_tasks/files/2664623/NECB2017simulations.zip)
JSON of NECB2017 baselines as created by NRC.  Note this is not using the latest method for NECB generation.

**Comment by IainMacdonald** [comment]
Not sure I understand what is in the NECB2017simulations.zip file.

**Comment by edwardv720** [comment]
> 
> 
> [NECB2017simulations.zip](https://github.com/canmet-energy/btap_tasks/files/2664623/NECB2017simulations.zip)
> JSON of NECB2017 baselines as created by NRC. Note this is not using the latest method for NECB generation.

Could you be more specific? What does that mean for the numerical results? Is there something we could do?

**Comment by phylroy** [comment]
I don't think you can do anything to help.   We are doing 2 things internally. 
1. We are revamping the autozoning and autosystem to be more realistic. So essentially there will be more control zone strategies being used as they would in real life while being true to NECB. 
2. We are staging the application of NECB so the create prototype building will be modified to run in stages and also pass data along the measures internally.

**Comment by phylroy** [comment]
Here are the remaining runs json output I ran on Amazon just now. Took < 1 hour to run and extract the data. 
Include opaque and fene runs. 


[edward.zip](https://github.com/canmet-energy/btap_tasks/files/2665141/edward.zip)

**Comment by phylroy** [comment]
> Not sure I understand what is in the NECB2017simulations.zip file.

@IainMacdonald  I already talked to Adam about this.  He said he was good with the json format.

**Comment by IainMacdonald** [comment]
Note from talking to Phylroy: file format is the same as qaqc.json and there is and example on the canmet/btap website. They are all in the same structure.

**Comment by edwardv720** [comment]
> 
> 
> Here are the remaining runs json output I ran on Amazon just now. Took < 1 hour to run and extract the data.
> Include opaque and fene runs.
> 
> [edward.zip](https://github.com/canmet-energy/btap_tasks/files/2665141/edward.zip)

Thanks Phylroy

**Comment by edwardv720** [comment]
Updated Opaque U-value improvement PAT proj - https://github.com/edwardv720/btap/tree/nrc_192/pat_projects/TG-BE%20envelope%20improvement%20(Uvalue)/N2017Op

**Comment by phylroy** [comment]
@edwardv720 @IainMacdonald they were so small that I just zipped everything up below..

Opaque 1
[results.6221f617-fc21-4ead-879f-9867e5c635e4.zip](https://github.com/canmet-energy/btap_tasks/files/2703602/results.6221f617-fc21-4ead-879f-9867e5c635e4.zip)

Opaque 2
[results.44b59b5e-f3c6-4e17-8c76-feae2348aaef.zip](https://github.com/canmet-energy/btap_tasks/files/2703672/results.44b59b5e-f3c6-4e17-8c76-feae2348aaef.zip)

Opaque 3
[results.d248db51-aa81-4f92-a72e-2cad29ef2cec.zip](https://github.com/canmet-energy/btap_tasks/files/2703700/results.d248db51-aa81-4f92-a72e-2cad29ef2cec.zip)

**Comment by IainMacdonald** [comment]
This was superseded by task 207
